### ISSUE TITLE
Won't load or create a level, just returns me to the main menu.

### ISSUE THREAD

**Issue by NickSProud** [jakimfett/Minechem#721]
When Ran with 
CoFHCore-3.0.0RC7-211
Minechem 5.0.5.394
MineFactory Reloaded 2.8.0 RC 8-86
ThermalDynamics 1.0.0 RC7-98
ThermalExpansion 4.0.0 RC7-141
ThermalFoundation 1.0.0RC7-62

almost forgot
Forge 10.13.3.1366-1.7.10

**Comment by stararker** [comment]
I got the same issue here is the actual exception you get. My guess is the cofh api is out of date. Could be wrong though.

```
java.lang.NullPointerException
	at net.minecraftforge.fluids.FluidRegistry.getFluidID(FluidRegistry.java:118) ~[FluidRegistry.class:?]
	at net.minecraftforge.fluids.FluidStack.getFluidID(FluidStack.java:112) ~[FluidStack.class:?]
	at cofh.thermalexpansion.util.crafting.TransposerManager.getFillRecipe(TransposerManager.java:41) ~[TransposerManager.class:?]
	at cofh.thermalexpansion.util.crafting.TransposerManager.fillRecipeExists(TransposerManager.java:51) ~[TransposerManager.class:?]
	at cofh.thermalexpansion.util.crafting.TransposerManager.addFillRecipe(TransposerManager.java:178) ~[TransposerManager.class:?]
	at cofh.thermalexpansion.util.crafting.TransposerManager.addFillRecipe(TransposerManager.java:223) ~[TransposerManager.class:?]
	at cofh.thermalexpansion.util.crafting.TransposerManager.loadRecipes(TransposerManager.java:101) ~[TransposerManager.class:?]
	at cofh.thermalexpansion.ThermalExpansion.loadComplete(ThermalExpansion.java:212) ~[ThermalExpansion.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_25]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_25]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:513) ~[forge-1.7.10-10.13.3.1362-1.7.10.jar:?]
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-16.0.jar:?]
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:208) ~[LoadController.class:?]
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:187) ~[LoadController.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_25]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_25]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-16.0.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-16.0.jar:?]
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:118) ~[LoadController.class:?]
	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:696) ~[Loader.class:?]
	at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:288) ~[FMLClientHandler.class:?]
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:541) ~[bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867) ~[bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) ~[1.7.10-Forge10.13.3.1362-1.7.10.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_25]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_25]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) ~[launchwrapper-1.11.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) ~[launchwrapper-1.11.jar:?]
[19:16:00] [Server thread/DEBUG] [FML/]: Reverting to frozen data state.
```

**Comment by way2muchnoise** [comment]
This is fixed as of #719 . You can get the new build of [Jenkins](http://jenkins.jakimfett.com/job/Minechem%20v5%20for%201.7.10/)

**Comment by NickSProud** [comment]
Got it working with that! Cheers!
